File: ed/go/examples/bench/fib.go
package fib

var s = make([]int, 0)

func Fib(n int) int {
	if n < 2 {
		s = append(s, n) // for fun!)
		return n
	}

	return Fib(n-1) + Fib(n-2)
}

File: ed/l/python/examples/whatever/fib.py
import time


def fib(n: int) -> int:
    time.sleep(0.1)
    if n < 2:
        return 1
    return fib(n - 1) + fib(n - 2)


def main(n: int) -> int:
    return fib(n)


print(main(5))
